traits/tests/test_traits.py

```diff
@@ -1095,7 +1095,7 @@ def test_trait_list_event(self):
         self.assertIs(self.last_event, old_event)
```

Comment:
Would this PR be a good place to update the `FIXME` comment in this test? It seems outdated.

Reply:
Yes! I'll do that. It makes documenting the new behaviour all the more important, though.

Reply:
and yes, the "wrapped in another list" bit is gone.

Reply:
Done.